internal/services/bot/bot_channel_directline_resource_test.go

```diff
@@ -101,7 +101,7 @@ resource "azurerm_bot_channel_directline" "test" {
   location            = "${azurerm_bot_channels_registration.test.location}"
   resource_group_name = "${azurerm_resource_group.test.name}"
   site {
-    name    = "test"
+    name    = "testSite"
```

Comment:
Do we have a specific reason the rename this?

Reply:
Reverted it.